taxonomies/categories.txt

```diff
@@ -48112,6 +48441,20 @@ fr:Crevettes
 it:Gamberetti
 nl:Garnalen
 
+<en:Shrimps
+en:Shrimp cooked
+fr:Crevette cuite
+ciqual_food_code:en:10007
+ciqual_food_name:en:Shrimp or prawn, cooked
+ciqual_food_name:fr:Crevette, cuite
+
+<en:Shrimps
+en:Shrimp raw
```

Comment:
For all categories below:

"XYZ raw" or "XYZ cooked" --> we need to put XYZ in plural, and put "Raw" or "Cooked" first.

```suggestion
en:Raw schrimps
```